public/index.html

```diff
@@ -1,11 +1,21 @@
 <!doctype html>
 <head>
   <title>o2znz</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css" />
 </head>
 <body>
   <div id="photo-gallery"></div>
+  <div id="checkout"></div>
```

Comment:
fyi I had to update the div name in modules to get them to match here (since they can't all be called 'app')

Reply:
these changes have been made on the individual modules now